tests/workflow/test_resolve_execution_config.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for the `_resolve_execution_config` helper function in PennyLane."""
```

Comment:
was `_resolve_execution_config` not unit tested before, when it lived in `workflow/qnode`?

Reply:
No, that was a recent helper function that was added. 😄  I probably should have added unit tests for it then but I'm doing it now.

Reply:
Hahah, fair enough 😅